repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/classloader/ClassLoaderTest.groovy

```diff
@@ -104,7 +95,6 @@ class ClassLoaderTest extends ReposeValveTest {
         then: "the request header should equal BAR"
         mc.handlings.size() == 1
         mc.receivedResponse.code == "200"
-        mc.handlings[0].request.headers.getFirstValue("FOO") == "BAR"
```

Comment:
Yeah, i considered that route, but it quickly spirals into more work than you would expect.